site/_posts/2019-10-01-implementing-fusionauth-python.md

```diff
@@ -593,26 +616,28 @@ ImportError: No module named fusionauth.fusionauth_client
 
 **What this usually means:** Do you have the FusionAuth-Client Library already installed? If so, did you execute the application outside of venv? Browse to the directory containing venv, and activate venv:
 
-```
-$ source venv/bin/activate
+```zsh
+source venv/bin/activate
 ```
 
 Then, change directories to your fusionapps folder and try again:
 
-```
-(venv) $ python test.py
+```zsh
+(venv) python test.py
```

Comment:
Remove `(venv)`

In general we don't want to show anything that the user might copy and paste incorrectly.

Reply:
fixed in latest commit